minikube/bootstrap.sh

```diff
@@ -91,6 +102,79 @@ upgradeHomebrewPackages() {
   done
 }
 
+# upgradeChocoPackages upgrades required Chocolatey packages to latest version.
+upgradeChocoPackages() {
+  runIfNot "choco list --local-only | grep 'minikube'" \
+    choco install minikube
```

Comment:
@targs08 before running `choco install` something it would be nice to update packages meta first, like we're doing in macOS `brew update`. Otherewise, there is a chance to work with outdated packages or skip new version upgrade.

Or Chocolatey doesn't have such thing as local meta info?

Reply:
@tyranron, Chocolatey doesn't have this feature.